src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import {useState} from 'react';
+
+export default function FormPage() {
+	const [invoice, setInvoice] = useState(false);
+	const [name, setName] = useState('');
+	const [amount, setAmount] = useState('');
+
+	return (
+		<div>
+			{invoice ? (
+				<form>
+					<ul>
+						<li>{name}</li>
+						<li>{amount}</li>
+					</ul>
+				</form>
```

Comment:
```suggestion
				<article>
					<ul>
						<li>{name}</li>
						<li>{amount}</li>
					</ul>
				</article>
```
You only ever use a form element if actually you want to get information of the user. In this specific case you only want to render the information that you already got. So an article is more fitting.

Reply:
Thank you, corrected it.